src/SonarQube.Client/Services/SonarQubeClient.cs

```diff
@@ -110,10 +133,10 @@ public Task<Result<PropertyDTO[]>> GetPropertiesAsync(ConnectionDTO connection,
                         request.QualityProfileKey, request.PageSize); // TODO: should handle optional params
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)